### ISSUE TITLE
Please add new GPG key on Keybase

### ISSUE THREAD

**Issue by henderea** [rvm/rvm#4525]
## Description

I no longer see a Keybase account to follow for the GPG key

## Steps to reproduce

 1. Look at the documentation

## Expected behavior

See a Keybase account I can follow to get the GPG key

## Actual behavior

No Keybase account listed.  Also, http://rvm.io/rvm/security should be updated with the new info

**Comment by obfuscoder** [comment]
Can we please get a fix for this? This breaks our build environment.

**Comment by tarellel** [comment]
This has bugged me for days, because at my work we have several different projects "all using rvm" with several different versions of ruby.

```shell
curl -sSL https://get.rvm.io | bash -s head
gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
rvm get head
```

**Comment by denisitpro** [comment]
Hi, I solved problem manual - use version 1.29.4 and disable check update

curl -sSL https://rvm.io/mpapis.asc | gpg2 --import -
/tmp/rvm-installer.sh 1.29.4 --path ~/.rvm --auto-dotfiles --user-install


disable update - its not true way (

**Comment by pkuczynski** [comment]
My key has been added to keybase and https://rvm.io/rvm/security has been updated with relevant info (check https://github.com/rvm/rvm-site/commit/edcb14091a350e49a57b84538d1192b0b1981397)
